apidef/oas/oas_test.go

```diff
@@ -254,7 +254,7 @@ func TestOAS_ExtractTo_ResetAPIDefinition(t *testing.T) {
 		"APIDefinition.Proxy.Transport.ProxyURL",
 		"APIDefinition.DisableQuota",
 		"APIDefinition.SessionLifetimeRespectsKeyExpiration",
-		"APIDefinition.SessionLifetime",
+		"APIDefinition.SessionLifetimeDisabled",
```

Comment:
As per the comment above this reads as a red flag; only a removal should be present here. There's a few set values before this block around L190 where SessionLifetimeDisabled should be set to true to remove it from the list.